### PR TITLE
Custom relayer

### PR DESCRIPTION
**Pull request by AshakaE**
closes https://github.com/EveripediaNetwork/issues/issues/2586

### CODE REVIEW

File: src/App/User/user.resolver.ts [hunk 1]
```diff
@@ -98,10 +98,13 @@ class UserResolver {
   }
 
   @ResolveField()
-  async wikis(@Parent() user: IUser, @Args() args: PaginationArgs) {
+  async wikis(@Parent() use: IUser, @Args() args: PaginationArgs) {
```
Comment (kesar) [inline src/App/User/user.resolver.ts:101]: why this change?

File: package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "@nestjs/typeorm": "^9.0.1",
     "@ntegral/nestjs-sentry": "^4.0.0",
     "@openzeppelin/defender-relay-client": "^1.54.1",
+    "@openzeppelin/defender-sdk": "^1.12.0",
```
Comment (kesar) [inline package.json:56]: why this is needed?

File: package.json [hunk 3]
```diff
@@ -74,7 +75,7 @@
     "cron": "^2.3.1",
     "dompurify": "^3.0.2",
     "esm": "^3.2.25",
-    "ethers": "^5.6.9",
+    "ethers": "^6.12.0",
```
Comment (kesar) [inline package.json:78]: lets ensure if we do a major update on ethers doesnt affect to other places where we might use it.. like new indexer and other places

Comment (AshakaE) [reply package.json:77]: I've updated all places it's being used, I'll keep an eye on it as we integrate the update
